Cant install

Funkosama opened the pull request:
Hello, im just a user, cant understad whats you mean " Extract the Zip folder to a preferred location. Run the Executable to enjoy adaptive triggers on Forza! " Whats folder, how run? can you please make detailed instructions with each step, preferably with screenshots, I bought a dualsenсe for forza, how wrong I was

cosmii02 committed:
Update appsettings.ini

The numbers there cannot be higher than 1.0, just saves time for some ppl I hope :(

cosmii02 committed:
Update appsettings.ini

Felt like the default ones are just always stiff and make the experience worse

cosmii02 committed:
Update appsettings.ini

Fixed the issue from previous commit

cosmii02 committed:
Update Program.cs

cosmii02 committed:
Huge progress but still barely beta

cosmii02 committed:
Better now

cosmii02 committed:
Everything done :)

cosmii02 committed:
UwU

cosmii02 committed:
readme update

cosmii02 committed:
readme typo fix

cosmii02 committed:
Fixed compatibility with dsx steam release while keeping github version working as well

cosmii02 committed:
Renaming stuff

cosmii02 committed:
Fixing stuff that I just broke

cosmii02 committed:
More stuff

cosmii02 committed:
Update Readme.md

cosmii02 committed:
Update Readme.md

cosmii02 committed:
Preparations to make old logic and config option again... idk why

cosmii02 committed:
Add files via upload

cosmii02 committed:
More preparations... still not sure why

cosmii02 committed:
More and more preparations.. great

cosmii02 committed:
Now just a few quick fixes to run and then should be pretty much all ready

cosmii02 committed:
Delete rigidversionfull directory

cosmii02 committed:
Update Program.cs

Changed a comment to reflect DSX name change

cosmii02 committed:
Update Readme.md

renaming more

Eemmpty commented:
I also tried to deal with this mod now, I did not understand what and how to install

cosmii02 committed:
Initial overhaul to instructions

Initial overhaul to instructions

Commit:
- Removing obj folder
- git ignore updated
- Triggers goes back to "normal state" when in menus (not racing)
- Started adding "vibration" feedback to throttle when losing grip

Commit:
- Updated throttle feedback calculation
- Added an extra "Is Race On" check since FH5 does not correctly update this value...

cosmii02 committed:
Merge pull request #5 from Chaoseiro/master

Throttle feedback, normal triggers for menus and cosmetic changes

cosmii02 committed:
Merge pull request #6 from cosmii02/V0.4.0

V0.4.0

Commit:
- Adjusts for drifting - checking front and rear tire slip separately
- Added global effect multiplier, allowing for effects to be disabled per trigger

Commit:
- A little better RPM scaling for the LED
- Added missing throttle threshold

Commit:
- New default values with smoother behavior overall

cosmii02 committed:
Update Program.cs

cosmii02 committed:
Merge pull request #7 from Chaoseiro/master

Adjusts for drifting & effect multiplier

cosmii02 committed:
Some more version tags to differentiate what version you are running

cosmii02 committed:
Create dotnet-desktop.yml

cosmii02 committed:
Update dotnet-desktop.yml

cosmii02 committed:
Delete dotnet-desktop.yml

cosmii02 committed:
Create dotnet.yml

cosmii02 committed:
Update dotnet.yml

cosmii02 committed:
Update dotnet.yml

cosmii02 committed:
Update dotnet.yml

cosmii02 committed:
Create devdotnet.yml

cosmii02 committed:
Update devdotnet.yml

cosmii02 committed:
Update devdotnet.yml

cosmii02 committed:
Update dotnet.yml

cosmii02 committed:
Update dotnet.yml

patmagauran commented:
Closing since @cosmii02 has their own repo for this now. https://github.com/cosmii02/ForzaDSX
